mllib/src/main/scala/org/apache/spark/ml/tree/treeParams.scala

```diff
@@ -493,13 +493,56 @@ private[ml] trait GBTParams extends TreeEnsembleParams with HasMaxIter {
 
   setDefault(maxIter -> 20, stepSize -> 0.1)
 
+  final val supportedStorageLevel: Array[String] =
```

Comment:
Revisiting this very old PR to say: you don't need all this. Use StorageLevel.fromString to parse the param.